components/MDX.tsx

```diff
@@ -258,7 +258,9 @@ export function getMDXComponents() {
         className={`${props.className || ''} border-b border-gray-200 px-6 py-4 text-sm leading-5 tracking-tight text-gray-700`}
       />
     ),
-    pre: (props: React.HTMLProps<HTMLPreElement>) => CodeComponent(props.children as CodeComponentProps),
+    pre: (props: React.HTMLProps<HTMLPreElement>) => {
+      return CodeComponent((props.children as React.ReactElement)?.props);
+    },
```

Comment:
Instead of using `return` in block, directly return the CodeComponent using arrow function.